skyportal/handlers/api/followup_request.py

```diff
@@ -385,6 +385,9 @@ def get(self, followup_request_id=None):
         except ValueError:
             return self.error("Invalid numPerPage value.")
 
+        if n_per_page > 1000:
+            return self.error('numPerPage should be no larger than 1000.')
+
```

Comment:
I'd also change the parsing of this variable to catch `TypeError` instead of checking if it is `None` (i.e., when input is `None` also return "Invalid page number value").